index.js

```diff
@@ -77,6 +77,12 @@ Prober.prototype.notok = function notok() {
     }
 };
 
+Prober.prototype.setEnabled = function setEnabled(enabled) {
+    if (typeof enabled === 'boolean') {
```

Comment:
assert is fine for programmer errors.

We should go for crash early instead of undefined behaviour.

People get paged when they make bugs, calling this with a non boolean is a bug.